pop-api/integration-tests/src/fungibles/utils.rs

```diff
@@ -309,21 +310,15 @@ pub(super) mod assets {
 	}
 
 	pub(crate) fn token_name(asset_id: AssetId) -> Vec<u8> {
-		<pallet_assets::Pallet<Runtime, TrustBackedAssetsInstance> as MetadataInspect<
-			AccountId32,
-		>>::name(asset_id)
+		Assets::name(asset_id)
```

Comment:
One line functions seem redundant.

Reply:
I prefer to keep it consistent and everything is called via the `assets` module

Reply:
I mean why wrap it in a function 'token_name()' in the utils pallet - its unnecessary complexity.

test() -> utils -> token_name() -> Assets::name()
vs
test() -> Assets::name()

Reply:
You are right, one liner helper functions do not make sense. Sorted in last commit